core/src/main/java/hudson/model/ListView.java

```diff
@@ -132,7 +132,7 @@ public void setJobFilters(List<ViewJobFilter> jobFilters) throws IOException {
         this.jobFilters.replaceBy(jobFilters);
     }
 
-    private Object readResolve() {
+    protected Object readResolve() {
```

Comment:
Does this imply that all plugins must make the same change?

Reply:
If the class is extended and the parent read resolve needs to be called

Reply:
https://github.com/jenkinsci/jep/tree/master/jep/228#inherited-private-serialization-methods explains this

Reply:
I attempted to search for extant plugins using an incorrect `private` override, but I might have missed some of course.